### ISSUE TITLE
Synchronization fails while having exception of recurring item in Outlook

### ISSUE THREAD

**Issue by ralfeus** [phw198/OutlookGoogleCalendarSync#702]
**OGCS Version**: v2.7.8.0
**Installed or Portable**: Portable

**Problem Description**  
Synchronization fails after adding recurrence exception. The error states: Failed to retrieve master for Google recurring event outside of sync range.

**Steps to Reproduce the Issue**  
1. Create a recurring item
2. Synchronize it
3. Add an exception inside synchronization time range
4. Try to synchronize again.

**Any other information**  
Log is here: https://gist.github.com/ralfeus/717cebc6009b4c31dde67309bda95295
```
2019-04-03 09:37:12,900 ERROR  19 OutlookGoogleCalendarSync.Console [0] -  Failed to retrieve master for Google recurring event outside of sync range.
2019-04-03 09:37:13,057 DEBUG  19 OutlookGoogleCalendarSync.Console [0] -  The following error was encountered during sync:-
2019-04-03 09:37:13,058 ERROR  19 OutlookGoogleCalendarSync.OGCSexception [0] -  System.Runtime.InteropServices.COMException: Operation aborted (Exception from HRESULT: 0x80004004 (E_ABORT))
2019-04-03 09:37:13,058 ERROR  19 OutlookGoogleCalendarSync.OGCSexception [0] -  Code: 0x80004004;-2147467260
2019-04-03 09:37:13,058 ERROR  19 OutlookGoogleCalendarSync.OGCSexception [0] -     at OutlookGoogleCalendarSync.Sync.Engine.synchronize()
   at OutlookGoogleCalendarSync.Sync.Engine.<>c__DisplayClass19_1.<Start>b__0(Object o, DoWorkEventArgs args)
2019-04-03 09:37:13,058 ERROR  19 OutlookGoogleCalendarSync.Console [0] -  Operation aborted (Exception from HRESULT: 0x80004004 (E_ABORT))
```

**Comment by ralfeus** [comment]
It turns out the problem is caused by reading AppointmentItem.Organizer on my work laptop. Not sure what could cause it. I tried to reproduce on a lab machine using non-admin user but couldn't. Probably it's Exchange policies prohibit getting this property. 

Made a patch to do not die on AppointmentItem.Organizer read failure

**Comment by phw198** [comment]
As you've noticed, this is the same underlying fix as for #697.

:package: I've taken a slightly different approach to resolving this - I'd appreciate it if you could try [hotfix v2.7.8.5.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/3077721/v2.7.8.5.zip) ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)) to see if it still works.

Please let me know how it goes, thanks.

**Comment by phw198** [comment]
[Reported as fixed](https://github.com/phw198/OutlookGoogleCalendarSync/issues/716#issuecomment-483549066) on #716
